search/query/boolean.go

```diff
@@ -61,41 +60,43 @@ func NewBooleanQueryMinShould(must []Query, should []Query, mustNot []Query, min
 
 // SetMinShould requires that at least minShould of the
 // should Queries must be satisfied.
-func (q *booleanQuery) SetMinShould(minShould float64) {
-	q.Should.(*disjunctionQuery).SetMin(minShould)
+func (q *BooleanQuery) SetMinShould(minShould float64) {
+	q.Should.(*DisjunctionQuery).SetMin(minShould)
 }
 
-func (q *booleanQuery) AddMust(m Query) {
+func (q *BooleanQuery) AddMust(m ...Query) {
```

Comment:
For folks that did used to have an slice of child queries...
qrys := []Query{q1, q2, q3, q4}

...might want to have an example somewhere that they can use the '...' idiom?
booleanQuery.AddMust(...qrys)